include/cetl/any.hpp

```diff
@@ -203,117 +358,63 @@ class any : private detail::base_move<Footprint, Copyable, Movable, Alignment>
         {
             if (rhs.has_value())
             {
-                any tmp;
-                rhs.handle(detail::action::Move, &tmp);
-                handle(detail::action::Move, &rhs);
-                tmp.handle(detail::action::Move, this);
+                any tmp{rhs};
+                rhs.copy_from(*this);
+                base::copy_from(tmp);
```

Comment:
Now your `copy_from` and `move_from` are no different from the standard copy/move assignment operators, why not invoke them directly here via `static_cast<base_copy&>`? This will allow you to make `copy_from` and `move_from` private (or remove them completely idk).

Reply:
>why not invoke them directly
yes, now I see (with help of `static_cast`) it's possible. previously I though that recursion will prevent it - haven't thought about casting